src/state/InitState.ts

```diff
@@ -67,10 +67,14 @@ export class InitState implements IState{
         // Check if docker is running and it's the correct version
         const isCorrectDockerComposeVersion = await this.dockerService.isCorrectDockerComposeVersion();
         const isDockerStarted = await this.dockerService.checkDocker();
+
+        await this.dockerService.checkDockerResources(this.cliOptions.multiNode);
+
```

Comment:
Can you make this exit using events like the ones below, so that StateController can properly catch it and do a cleanup afterwards?